src/render/bsdf.cpp

```diff
@@ -40,6 +40,11 @@ MI_VARIANT Spectrum BSDF<Float, Spectrum>::eval_diffuse_reflectance(
     return eval(ctx, si, wo, active) * dr::Pi<Float>;
 }
 
+MI_VARIANT Float BSDF<Float, Spectrum>::eval_roughness(
+    const SurfaceInteraction3f & /* si */, Mask /* active */) const {
+    return 1.f;
```

Comment:
Let's have the default value be 0.

Reply:
Wouldn't it cause wierd behavior for the downstream applications of this method? Isn't the default roughness for a diffuse surface is considered 1?

I am assuming if someone is using this function to input it to a neural net as roughness, they would want to say 1 instead of 0.0,  cause roughness=0.0 mean an extremely glossy surface.

Reply:
Indeed, I agree.